src/mocks.ts

```diff
@@ -533,11 +532,9 @@ const globals = {
         tmpState.set(key, value);
       }
 
-      const val = JSON.stringify(value);
-
-      IOGas = IOGas + val.length; // Total serialized length of gas
+      IOGas = IOGas + JSON.stringify(value).length; // Total serialized length of gas
 
-      return val ?? "null"; //insta.exports.__newString(val);
```

Comment:
This does not match the production implementation. Copy [these lines](https://github.com/vsc-eco/vsc-node/blob/main/src/services/new/vm/vm-runner.ts#L904L908) and leave the `insta.exports.__newString(val)` commented out.

Reply:
the prod implementation in here results empty keys to be returned as the string "null". 
concretly 
fetchedEmptyKey === "null" -> false
fetchedEmptyKey === '"null"' -> true

which is why i changed it for this test environment

@Geo25rey

Reply:
this is different for the prod setup

Reply:
in the prod setup 
fetchedEmptyKey === "null" -> true